pkg/operator/baremetal_test.go

```diff
@@ -3,10 +3,37 @@ package operator
 import (
 	"testing"
 
+	"sigs.k8s.io/yaml"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	fakedynamic "k8s.io/client-go/dynamic/fake"
 	fakekube "k8s.io/client-go/kubernetes/fake"
 )
 
+var yamlContent = `
+apiVersion: metal3.io/v1alpha1
+kind: Provisioning
+metadata:
+  name: test
+  labels:
```

Comment:
please remove this label, it has no meaning

Reply:
Done.